components/dialogs/MissionDialog.vue

```diff
@@ -134,6 +136,11 @@
           </div>
           <!-- END - Invite Token Sale Section -->
         </div>
+        <mission-completed-banner
```

Comment:
put `v-if="isCompleted"` in front, then `v-else` for main body seems more straight forward?

Reply:
@williamchong007 Updated